unofficial/c511027103.lua

```diff
@@ -0,0 +1,104 @@
+    e1:SetProperty(EFFECT_FLAG_CARD_TARGET)
+    e1:SetCategory(CATEGORY_EQUIP)
+    e1:SetRange(LOCATION_HAND+LOCATION_MZONE)
+    e1:SetTarget(s.eqtg)
+    e1:SetOperation(s.eqop)
+    c:RegisterEffect(e1)
+    --protecc
+    local e2=Effect.CreateEffect(c)
+    e2:SetType(EFFECT_TYPE_SINGLE)
+    e2:SetProperty(EFFECT_FLAG_SINGLE_RANGE)
+    e2:SetCode(EFFECT_CANNOT_BE_EFFECT_TARGET)
+    e2:SetRange(LOCATION_SZONE)
+    e2:SetCondition(s.con)
+    e2:SetValue(s.val)
+    c:RegisterEffect(e2)
+    --destroy replace
+    local e3=Effect.CreateEffect(c)
+    e3:SetType(EFFECT_TYPE_FIELD+EFFECT_TYPE_CONTINUOUS)
+    e3:SetCode(EFFECT_DESTROY_REPLACE)
+    e3:SetRange(LOCATION_SZONE)
+    e3:SetTarget(s.reptg)
+    e3:SetValue(s.repval)
+    e3:SetOperation(s.repop)
+    c:RegisterEffect(e3)
+end
+
+s.listed_names={56840427}
+
+--equip
+
+function s.eqtg(e,tp,eg,ep,ev,re,r,rp,chk,chkc)
+	if chkc then return chkc:IsLocation(LOCATION_MZONE) and chkc:IsControler(tp) and chkc:IsFaceup() and chkc:IsCode(56840427) end
+	if chk==0 then return Duel.GetLocationCount(tp,LOCATION_SZONE)>0
+		and Duel.IsExistingTarget(aux.FilterFaceupFunction(Card.IsCode,56840427),tp,LOCATION_MZONE,0,1,nil) end
+	Duel.Hint(HINT_SELECTMSG,tp,HINTMSG_EQUIP)
+	Duel.SelectTarget(tp,aux.FilterFaceupFunction(Card.IsCode,56840427),tp,LOCATION_MZONE,0,1,1,nil)
+	Duel.SetOperationInfo(0,CATEGORY_EQUIP,e:GetHandler(),1,0,0)
+end
+
+function s.eqop(e,tp,eg,ep,ev,re,r,rp)
+	local c=e:GetHandler()
+	if not c:IsRelateToEffect(e) then return end
+	if c:IsLocation(LOCATION_MZONE) and c:IsFacedown() then return end
+	local tc=Duel.GetFirstTarget()
+	if Duel.GetLocationCount(tp,LOCATION_SZONE)<=0 or tc:GetControler()~=tp or tc:IsFacedown() or not tc:IsRelateToEffect(e) then
+		Duel.SendtoGrave(c,REASON_EFFECT)
+		return
+	end
+	Duel.Equip(tp,c,tc,true)
+	local e1=Effect.CreateEffect(c)
+	e1:SetType(EFFECT_TYPE_SINGLE)
+	e1:SetCode(EFFECT_EQUIP_LIMIT)
+	e1:SetReset(RESET_EVENT+RESETS_STANDARD)
+	e1:SetValue(s.eqlimit)
+	e1:SetLabelObject(tc)
+	c:RegisterEffect(e1)
+	--atkup
+	local e2=Effect.CreateEffect(c)
+	e2:SetType(EFFECT_TYPE_EQUIP)
+	e2:SetCode(EFFECT_UPDATE_ATTACK)
+	e2:SetValue(1200)
+	e2:SetReset(RESET_EVENT+RESETS_STANDARD)
+	c:RegisterEffect(e2)
+end
+
+function s.eqlimit(e,c)
+	return c==e:GetLabelObject()
+end
+
+--protecc
+
+function s.con(e,tp,eg,ep,ev,re,r,rp)
+	return e:GetHandler():GetEquipTarget()~=nil
+end
+
+function s.val(e,re)
+	return re:IsHasCategory(CATEGORY_DESTROY)
+end
+
+--desrep
+
+function s.reptg(e,tp,eg,ep,ev,re,r,rp,chk)
+    if chk==0 then return e:GetHandler():IsAbleToHand() and eg:IsExists(s.repfilter,1,nil,e) end
+    return Duel.SelectEffectYesNo(tp,e:GetHandler(),96)
+end
+function s.repval(e,c)
+    return s.repfilter(c,e)
+end
+function s.repop(e,tp,eg,ep,ev,re,r,rp)
+    Duel.SendtoHand(e:GetHandler(),nil,REASON_EFFECT+REASON_REPLACE)
+end
+
+function s.repfilter(c,e)
+    return c==e:GetHandler():GetEquipTarget() and c:IsReason(REASON_EFFECT)

```

Comment:
You don't need ```and c:IsReason(REASON_EFFECT)```, but you do need ```and not c:IsReason(REASON_REPLACE)```. Also remember to backport it to Tornado Bringer (anime)